moto/ssm/utils.py

```diff
@@ -0,0 +1,4 @@
+def parameter_arn(region, parameter_name):
+    if parameter_name[0] == "/":
+        parameter_name = parameter_name[1:]
+    return "arn:aws:ssm:{0}:1234567890:parameter/{1}".format(region, parameter_name)
```

Comment:
Please pull out the account ID into a separate constant at the top of the file. This makes it easy to mock out tests where account ID's matter.

Reply:
Hum this is not an util for tests though. I based this code on `secret_arn` of the secrets manager.

Reply:
You can use it in your tests by doing something along the lines of:
`mock.patch("moto.ssm.models.ACCOUNT_ID", "222222222222")`
^^ This would then return ARNs with the account ID portion set to `222222222222`.

Reply:
Well it is fixed, I have not idea why the tests fail though. Not related to this PR.

Reply:
Make sense! It is fixed, but the tests fail for some reason not related to this PR.

Reply:
I'll take a look at what happened with they xray tests.